notebooks/ivf_flat_example.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
Not sure if this change is valid. Do you mind looking at this notebook real quick just to verify this isn't stale?

Reply:
I just reverted the changes.